### PR TITLE
[MRG] design- and text-corrections

### PR DESCRIPTION
**Pull request by michelfernandez**
- Changed config.yaml - some left had no "/" at the beginning. So the active pages in the submenu were not excellent
- Change Logo Canton FR and Text to State in DE, EN, FR for the pages index and features
- Changed images in contact with better resolution
- Changed legal-weight images - text changes - only in EN and FR
- Removed why qes from legal requirements in DE, EN, FR for index, hr and lawyer
- Add Use Skribble to maximise the legal weight of your signature for index, hr and lawyer
- Changed title for get qes in DE, FR, EN

### CODE REVIEW

File: site/content/english/_index.md [hunk 1]
```diff
@@ -143,79 +137,32 @@ Skribble and its signing partner Swisscom commit to manage data in accordance wi
...
+  href="/signaturestandards/"
   outlined="true"
 %}}
 
-{{< picture AES 100 "Signed with Skribble means legally signed according to Swiss and EU law." >}}
-
 {{% richtext %}}
-**Signing with AES**<br>
-For the advanced electronic signature (AES) no identification is necessary.
+**At Skribble you have the choice <br class="hide-for-mobile">between the signature standards FES and QES.**<br>
```
Comment (betatim) [inline site/content/english/_index.md]:
```suggestion
**At Skribble you have the choice <br class="hide-for-mobile">between the signature standards AES and QES.**<br>
```
I don't think we use FES in english

File: site/content/english/lawyer.md [hunk 2]
```diff
@@ -106,26 +106,35 @@ Skribble and its signing partner Swisscom commit to manage data in accordance wi
...
-  target="_blank"
+  href="/signaturestandards/"
   outlined="true"
 %}}
 
-{{< picture qes-erklaert 122 "Signed with Skribble means legally signed according to Swiss and EU law." >}}
-
 {{% richtext %}}
-**Why does the qualified electronic signature (QES) sets a digital standard by the law?**<br>
+**At Skribble you have the choice <br class="hide-for-mobile">between the signature standards FES and QES.**<br>
```
Comment (betatim) [inline site/content/english/lawyer.md]:
```suggestion
**At Skribble you have the choice <br class="hide-for-mobile">between the signature standards AES and QES.**<br>
```

**Comment by michelfernandez** [comment]
Thanks!

**Comment by michelfernandez** [comment]
@betatim ist ready to merge now! Thanks!

**Pull request merged**